style.css

```diff
@@ -438,3 +457,283 @@ a {
   margin-bottom: 30px;
   text-align: center;
 }
+
+/* The desktop design */
+@media (min-width: 650px) {
+  .hamburger {
+    display: none;
+  }
+
+  .nav-bar {
```

Comment:
- [ ] Please make sure that, as per requirement, the desktop view is triggered in the range of 992px - 1100px. Currently, yours sits at 650px, which is too low.